scipy/fft/_pocketfft/tests/test_real_transforms.py

```diff
@@ -194,8 +202,17 @@ def test_complex(transform, dtype):
     assert_array_almost_equal(x, y)
 
 
+if TYPE_CHECKING:
```

Comment:
This line shouldn't be here. It causes `DecMapType` to not be defined at runtime, which causes the failures in CI:
```
NameError: name 'DecMapType' is not defined
```
In general you only want to use `if TYPE_CHECKING` is something does not work or is expensive at runtime.